ecs-init/config/os_family_linux_test.go

```diff
@@ -0,0 +1,136 @@
+//go:build unit
+// +build unit
+
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package config
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	expectedDebianOSFamily = "debian_11"
+
+	// Base OS release content without ID/VERSION_ID lines
+	debianOSReleaseBase = `PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
+NAME="Debian GNU/Linux"
```

Comment:
Have you manually tested it against an Amazon Linux instance? Just want to make sure its all good with ECS AMIs, that majority of our customers would use.